packages/jest-cli/src/test_scheduler.js

```diff
@@ -96,6 +96,23 @@ export default class TestScheduler {
         });
         return Promise.resolve();
       }
+      if (testResult.leaks) {
+        const message = [
+          'Your test suite is leaking memory! Please ensure all references are cleaned.',
+          '',
+          'There is a number of things that can leak memory:',
+          '  - Async operations that have not finished (e.g. fs.readFile).',
+          '  - Timers not properly mocked (e.g. setInterval, setTimeout).',
+          '  - Missed global listeners (e.g. process.on).',
+          '  - Keeping references to the global scope.',
+        ].join('\n');
```

Comment:
Can we not use an array and just use backticks + string concatenation like in the rest of Jest? :)